docs-kits/kits/Modular Production Kit/page_adoption-view.md

```diff
@@ -84,5 +84,5 @@ The standards for release 3.2 will be published soon. Our relevant standards can
 This work is licensed under the [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/legalcode).
 
 - SPDX-License-Identifier: CC-BY-4.0
-- SPDX-FileCopyrightText: 2023,Fraunhofer Institute of Optronics, System Technology and Image Exploitation (IOSB)
+- SPDX-FileCopyrightText: 2023;Fraunhofer-Gesellschaft zur Foerderung der angewandten Forschung e.V. (represented by Fraunhofer IWU & Fraunhofer IOSB & Fraunhofer IPA)
```

Comment:
```suggestion
- SPDX-FileCopyrightText: 2023 Fraunhofer-Gesellschaft zur Foerderung der angewandten Forschung e.V. (represented by Fraunhofer IWU & Fraunhofer IOSB & Fraunhofer IPA)
```

following the tips and tricks on one of your  [TRG 7-7](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-07#documentation)